test/manual/search-index-management.prose.test.ts

```diff
@@ -0,0 +1,210 @@
+          predicate: indexes => indexes.every(index => index.queryable),
+          indexName: 'test-search-index'
+        });
+
+        expect(index).to.exist;
+        expect(index)
+          .to.have.property('latestDefinition')
+          .to.deep.equal({ mappings: { dynamic: false } });
+      }
+    );
+
+    it(
+      'Case 2: Driver can successfully create multiple indexes in batch',
+      metadata,
+      async function () {
+        const indexDefinitions = [
+          {
+            name: 'test-search-index-1',
+            definition: {
+              mappings: { dynamic: false }
+            }
+          },
+          {
+            name: 'test-search-index-2',
+            definition: {
+              mappings: { dynamic: false }
+            }
+          }
+        ];
+
+        const names = await collection.createSearchIndexes(indexDefinitions);
+        expect(names).to.deep.equal(['test-search-index-1', 'test-search-index-2']);
+
+        const indexes = await waitForIndexes({
+          predicate: indexes => indexes.every(index => index.queryable)
+        });
+
+        for (const indexDescription of indexDefinitions) {
+          const index = indexes.find(({ name }) => name === indexDescription.name);
+          expect(index, `expected ${indexDescription.name} to exist`).to.exist;
+
+          expect(index)
+            .to.have.property('latestDefinition')
+            .to.deep.equal({ mappings: { dynamic: false } });
+        }
+      }
+    );
+
+    it('Case 3: Driver can successfully drop search indexes', metadata, async function () {
+      const name = await collection.createSearchIndex({
+        name: 'test-search-index',
+        definition: {
+          mappings: { dynamic: false }
+        }
+      });
+
+      expect(name).to.equal('test-search-index');
+
+      await waitForIndexes({
+        predicate: indexes => indexes.every(index => index.queryable),
+        indexName: 'test-search-index'
+      });
+
+      await collection.dropSearchIndex('test-search-index');
+
+      const indexes = await waitForIndexes({
+        predicate: indexes => indexes.length === 0,
+        indexName: 'test-search-index'
+      });
+
+      expect(indexes).to.deep.equal([]);
+    });
+
+    it('Case 4: Driver can update a search index', metadata, async function () {
+      const name = await collection.createSearchIndex({
+        name: 'test-search-index',
+        definition: {
+          mappings: { dynamic: false }
+        }
+      });
+
+      expect(name).to.equal('test-search-index');
+
+      await waitForIndexes({
+        predicate: indexes => indexes.every(index => index.queryable),
+        indexName: 'test-search-index'
+      });
+
+      await collection.updateSearchIndex('test-search-index', { mappings: { dynamic: true } });
+
+      const [updatedIndex] = await waitForIndexes({
+        predicate: indexes => indexes.every(index => index.queryable && index.status === 'READY'),
```

Comment:
The test on `index.status === 'READY'` is not in the spec and doesn't seems necessary since `queryable` is `false` where `status` is `PENDING` after an update.

```suggestion
        predicate: indexes => indexes.every(index => index.queryable),
```